core/ui/EditTemplate/body/default.tid

```diff
@@ -18,7 +18,7 @@ $:/config/EditorToolbarButtons/Visibility/$(currentTiddler)$
 	importState=<<qualify $:/state/ImportImage>> >
 <$dropzone importTitle=<<importTitle>> autoOpenOnImport="no" contentTypesFilter={{$:/config/Editor/ImportContentTypesFilter}} class="tc-dropzone-editor" enable={{{ [{$:/config/DragAndDrop/Enable}match[no]] :else[subfilter{$:/config/Editor/EnableImportFilter}then[yes]else[no]] }}} filesOnly="yes" actions=<<importFileActions>> >
 <div>
-<div class={{{ [function[edit-preview-state]match[yes]then[tc-tiddler-preview]] +[join[ ]] }}}>
+<div class={{{ [function[edit-preview-state]match[yes]then[tc-tiddler-preview]else[tc-no-tiddler-preview]] [[tc-grid]] +[join[ ]] }}}>
```

Comment:
We try to use semantic class names, and avoid functional names such as "tc-grid". I think it would be clearer to replace `tc-no-tiddler-preview` with `tc-tiddler-preview-hidden`, and to use `tc-tiddler-preview-frame` instead of `tc-grid`.

Reply:
I think `tc-tiddler-preview-frame` is way to specific. `tc-grid` only contains 1 setting to use grid. It can be reused that way. 

`tc-tiddler-preview-frame` is completely misleading since it is not set to the preview-frame, but it's parent

Reply:
If I want to define `tc-grid` to any other gird with a completely different context in the sidebar or control-panel `tc-tiddler-preview-frame` would be misleading. IMO it should be as generic as possible

Reply:
@pmario if we use `tc-grid` and reuse it elsewhere then we'll be constrained that we can't change the styling of just the preview pane. As I said before, functional class names are an anti-pattern.